.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release SPRAS on new tags
+# This action runs whenever a new tag matching vX.Y.Z is pushed to the repo.
+# Its job is to build the `spras` container and push it to DockerHub with the
+# 'latest' tag, and a tag matching the one supplied to GitHub.
+
+on:
+  push:
+    tags:
+      # run release on any version tag vX.Y.Z, e.g. v0.2.1
+      - v[0-9]+\.[0-9]+\.[0-9]+
```

Comment:
Our [tags](https://github.com/Reed-CompBio/spras/tags) so far have not had a leading v. Should we keep that convention and change the pattern here?

Reply:
Fixed in the latest commit.